docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -237,8 +235,39 @@ function getAddress(first, second, invocation) {
 }
 ```
 
-<a id="returns"></a>
+<a id="requiresParameterAddresses"></a>
+### @requiresParameterAddresses
+
+Indicates that the function should return the addresses of input parameters. 
 
+The last function parameter must be of type `CustomFunctions.Invocation` or a derived type to use  `@requiresParameterAddresses`. The JSDoc must also include an `@returns` tag specifying that the return value be a matrix, such as `@returns {string[][]}` or `@returns {number[][]}`. See [Matrix types](/office/dev/add-ins/excel/custom-functions-json-autogeneration#matrix-type) for additional information. 
```

Comment:
```suggestion
 The JSDoc must also include an `@returns` tag specifying that the return value be a matrix, such as `@returns {string[][]}` or `@returns {number[][]}`. 
```
I'm kinda wondering if JSDoc is the right name for where the tags go?

Reply:
Discussed offline -- I will adjust the JSDoc phrasing.